src/index.js

```diff
@@ -1,16 +1,7 @@
-const express = require('express');
-const Server = require('./my-server');
-
-const app = express();
-
-app.use(express.static('static'));
+const App = require('./app');
 
 if (require.main === module) {
-    const server = new Server(app);
+    const app = new App({ port: 3002 });
```

Comment:
If we hard-code the port value, we will lose the option to control it via the `.env` files and environment variables, don't we?

Reply:
Yes, but I thought that `.env` configuration will be introduced in future pull request. Anyway, hard-coded port seems really bad.